lib/internal/event_target.js

```diff
@@ -44,6 +45,9 @@ class Event {
 
 
   constructor(type, options) {
+    if (arguments.length === 0) {
+      throw new ERR_INVALID_ARG_COUNT(1, arguments.length);
```

Comment:
We already have `ERR_MISSING_ARGS`. It has the same intention.

Reply:
Thank you @BridgeAR 😆 ... I could have sworn we already had something but couldn't remember what it was and didn't have the opportunity to check on it.

Reply:
Good catch

Reply:
Refactored to use ERR_MISSING_ARGS